plugins/typed-variable-modal/package.json

```diff
@@ -41,15 +41,15 @@
   ],
   "devDependencies": {
     "@blockly/dev-scripts": "^1.2.10",
-    "@blockly/dev-tools": "^2.6.1",
-    "blockly": "^3.20200625.0",
+    "@blockly/dev-tools": "^3.0.0",
+    "blockly": "^7.20211209.0",
     "jsdom": "^19.0.0",
     "jsdom-global": "3.0.2",
     "mocha": "^7.1.0",
     "sinon": "7.5.0"
   },
   "peerDependencies": {
-    "blockly": "3.20200625.0 - 6"
+    "blockly": "3.20200625.0 - 7"
```

Comment:
Good catch, therefore gave this one a new major version as well. And updated the dependency on plugin-modal to be the newest version so that the peerDependency on blockly is satisfied on both.